upgrade_test.py

```diff
@@ -736,6 +736,87 @@ def test_generic_cluster_upgrade(self):  # pylint: disable=too-many-locals,too-m
         stress_after_cluster_upgrade_pool = self.run_stress_thread(stress_cmd=stress_after_cluster_upgrade)
         self.verify_stress_thread(stress_after_cluster_upgrade_pool)
 
+    def test_alternator_upgrade(self):
```

Comment:
the only difference between this function and and `test_generic_cluster_upgrade()`, are the name of from the SCT config.
please remove this function. and use `test_generic_cluster_upgrade()`

Reply:
Done